backoffice/src/views/delete-record/delete-record.scss

```diff
@@ -0,0 +1,7 @@
+.delete-beacon-container {
+  padding: 2% 2% 2%;
```

Comment:
Might be worth pulling the form as a child of this, I know it's unlikely there will be another .delete-beacon-form but it's good practice.